### PR TITLE
DIV-5521 - added config files to divorce CMS aat/perftest

### PR DESCRIPTION
**Pull request by hosainnet**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines

Linked to https://github.com/hmcts/div-case-maintenance-service/pull/169

### JIRA link (if applicable) ###

DIV-5521

### Change description ###

For AKS migration

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/perftest/common/divorce/div-cms.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      applicationInsightsInstrumentKey: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
+      image: hmctspublic.azurecr.io/div/cms:prod-f7412e3e
+      ingressHost: div-cms-perftest.service.core-compute-perftest.internal
+      environment:
+        AUTH_PROVIDER_SERVICE_CLIENT_BASEURL : "http://rpe-service-auth-provider-perftest.service.core-compute-perftest.internal"
+        DRAFT_STORE_API_BASEURL : "http://draft-store-service-perftest.service.core-compute-perftest.internal"
+        IDAM_CASEWORKER_USERNAME : "DivCaseWorker@AAT.com"
+        CASE_FORMATTER_SERVICE_API_BASEURL : "http://div-cfs-perftest.service.core-compute-perftest.internal"
+        IDAM_API_BASEURL: "https://idam-api.aat.platform.hmcts.net"
+        IDAM_API_REDIRECT_URL : "https://div-pfe-perftest.service.core-compute-perftest.internal/authenticated"
```
Comment (hosainnet) [inline k8s/perftest/common/divorce/div-cms.yaml]: Question, do these `perftest` have to exist at this point?

File: k8s/aat/common/divorce/div-cms.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      environment:
+        AUTH_PROVIDER_SERVICE_CLIENT_BASEURL : "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+        DRAFT_STORE_API_BASEURL : "http://draft-store-service-aat.service.core-compute-aat.internal"
+        IDAM_CASEWORKER_USERNAME : "DivCaseWorker@AAT.com"
+        CASE_FORMATTER_SERVICE_API_BASEURL : "http://div-cfs-aat.service.core-compute-aat.internal"
+        IDAM_API_BASEURL: "https://idam-api.aat.platform.hmcts.net"
+        IDAM_API_REDIRECT_URL : "https://div-pfe-aat.service.core-compute-aat.internal/authenticated"
+        CASE_DATA_STORE_BASEURL : "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
+        MANAGEMENT_ENDPOINT_HEALTH_CACHE_TIMETOLIVE: "30000"
+        DOCUMENTATION_SWAGGER_ENABLED: "true"
```
Comment (hosainnet) [inline k8s/aat/common/divorce/div-cms.yaml]: Copied the above from (Apart from reform env and app insights) https://github.com/hmcts/div-case-maintenance-service/blob/master/charts/div-cms/values.aat.template.yaml

File: k8s/perftest/common/divorce/div-emca.yaml [hunk 3]
```diff
@@ -41,6 +41,6 @@ spec:
         enabled: true
     global:
       environment: perftest
-      subscriptionId: "eddbcec4-6ebc-4507-9f22-38d53a92e761"
```
Comment (hosainnet) [inline k8s/perftest/common/divorce/div-emca.yaml]: This was div-perftest appinsights ID which I assume is wrong, I've change it to the ID of the `DCD-CNP-QA` subscription which is where `perftest` is

File: k8s/perftest/common/divorce/div-emca.yaml [hunk 4]
```diff
@@ -19,7 +19,7 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      applicationInsightsInstrumentKey: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
+      applicationInsightsInstrumentKey: "eddbcec4-6ebc-4507-9f22-38d53a92e761"
```
Comment (hosainnet) [inline k8s/perftest/common/divorce/div-emca.yaml]: Screenshot 2019-08-22 at 15 05 53

File: k8s/aat/common/divorce/div-cms.yaml (hunk 2 above)
Comment (qzhou-hmcts) [reply k8s/aat/common/divorce/div-cms.yaml]: this should be false for prod and overridden on aat

File: k8s/perftest/common/divorce/div-cms.yaml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "eddbcec4-6ebc-4507-9f22-38d53a92e761"
+      image: hmctspublic.azurecr.io/div/cms:prod-f7412e3e
+      ingressHost: div-cms-perftest.service.core-compute-perftest.internal
+      environment:
+        AUTH_PROVIDER_SERVICE_CLIENT_BASEURL : "http://rpe-service-auth-provider-perftest.service.core-compute-perftest.internal"
+        DRAFT_STORE_API_BASEURL : "http://draft-store-service-perftest.service.core-compute-perftest.internal"
+        IDAM_CASEWORKER_USERNAME : "DivCaseWorker@AAT.com"
+        CASE_FORMATTER_SERVICE_API_BASEURL : "http://div-cfs-perftest.service.core-compute-perftest.internal"
+        IDAM_API_BASEURL: "https://idam-api.aat.platform.hmcts.net"
```
Comment (qzhou-hmcts) [inline k8s/perftest/common/divorce/div-cms.yaml]: should this be perftest too?

File: k8s/aat/common/divorce/div-cms.yaml (hunk 2 above)
Comment (hosainnet) [reply k8s/aat/common/divorce/div-cms.yaml]: Yeah, this is the AAT file so we're overriding it

File: k8s/perftest/common/divorce/div-cms.yaml (hunk 5 above)
Comment (hosainnet) [reply k8s/perftest/common/divorce/div-cms.yaml]: Yep good spot thanks

Comment (hosainnet) [reply k8s/perftest/common/divorce/div-cms.yaml]: @qzhou-hmcts  https://github.com/hmcts/cnp-flux-config/pull/482

**Pull request merged**